clients/python/llmengine/model.py

```diff
@@ -72,41 +73,41 @@ def create(
                 Source of the LLM. Currently only HuggingFace is supported
 
             inference_framework (`LLMInferenceFramework`):
-                Inference framework for the LLM. Currently only DeepSpeed is supported
+                Inference framework for the LLM. Currently supported frameworks are DeepSpeed, text-generation-inference, vLLM, and LightLLM
 
             num_shards (`int`):
                 Number of shards for the LLM. When bigger than 1, LLM will be sharded
                 to multiple GPUs. Number of GPUs must be equal or larger than num_shards.
-                Only affects behavior for text-generation-inference models
 
             quantize (`Optional[Quantization]`):
-                Quantization for the LLM. Only affects behavior for text-generation-inference models
+                Quantization method for the LLM. Different frameworks support different set of quantization methods.
 
             checkpoint_path (`Optional[str]`):
-                Path to the checkpoint for the LLM. For now we only support loading a tar file from AWS S3.
+                Path to the checkpoint for the LLM. Can be either a folder (preferred since there's no untar) or a tar file.
```

Comment:
> Can be either a folder

I think we want to be precise and consistent around our guidance relating to remote files/directories. Would suggest cross-checking with the Files API and making sure that the explanation around this makes sense. e.g. we may want to clarify that this is meant to be some remote path that's accessible by the LLM Engine deployment, etc.

> Can be either a folder (preferred since there's no untar)
Might be good to explain why skipping the untar is desirable - is it cold start time?

Reply:
checking code, i don't think we only support creating endpoints from files created with Files API.

yes skipping untar is good to cold start time